docs/Backend/01. backend-convention.md

```
@@ -25,11 +25,16 @@
 
 ## 5) 도메인/에러 규칙
 - 도메인 객체 생성은 `@Builder`를 사용하고, Builder로 생성한다.
-- 에러 코드는 도메인별로 커스텀 `*ErrorCode`를 작성한다.
+- 에러 코드는 도메인별로 커스텀 `*ErrorCode` enum을 작성한다.
 - 에러 코드는 짧고 명확하게 유지한다.
+- 각 도메인은 전용 Exception 클래스(`*Exception`)를 작성하여 사용한다.
+  - 모든 커스텀 예외는 `com.web.SearchWeb.config.exception.BusinessException`을 상속받는다.
+  - 에러 코드 인터페이스: `com.web.SearchWeb.config.exception.ErrorCode`
+  - 예시: `linkanalysis` 도메인 → `LinkAnalysisException`
 
 ## 6) Config 배치 규칙
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)